src/main.ts

```diff
@@ -3,3 +3,7 @@ import { createApp } from 'vue'
 import { App } from './app'
 
 createApp(App).mount('#app')
+
+if ('serviceWorker' in navigator) {
```

Comment:
Поскольку Passcryptum могут запускать из файловой системы, надо ещё проверять `location.host`. Тогда в случае запуска из файла, SW не будет регистрироваться.

```suggestion
if (location.host && 'serviceWorker' in navigator) {
```